src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureSupplier, AreaCalculator {
+    private String color;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public abstract void drawFigure();
```

Comment:
```suggestion
    public abstract void draw();
```
let's avoid redundancy in method names